### PR TITLE
Add seasonal strength test

### PR DESCRIPTION
**Pull request by iagochavarry**
Created seasonal strength test
Add test and seasonal_test parameters in the auto_arima function
Add the possibility to fix d or D in the auto_arima function

### CODE REVIEW

File: src/models/sarima.jl [hunk 1]
```diff
@@ -878,11 +907,17 @@ function auto_arima(y::Vector{Fl};
...
-                    show_trace::Bool = false
+                    show_trace::Bool = false,
+                    test::String = "kpss",
+                    seasonal_test::String = "seas"
                     ) where Fl <: AbstractFloat
     
     # Assert parameters
     @assert seasonal >= 0
+    @assert D >= -1
+    @assert D <= max_D
```
Comment (guilhermebodin) [inline src/models/sarima.jl:918]: Is there a max_D and max_d?

File: src/models/sarima.jl [hunk 2]
```diff
@@ -878,11 +907,17 @@ function auto_arima(y::Vector{Fl};
                     max_order::Int = 5,
                     information_criteria::String = "aicc",
                     allow_mean::Bool = true,
-                    show_trace::Bool = false
+                    show_trace::Bool = false,
+                    test::String = "kpss",
+                    seasonal_test::String = "seas"
```
Comment (guilhermebodin) [inline src/models/sarima.jl:912]: test and seasonal test are not good names. What do you think about integration test and seasonal integration test.

File: src/models/sarima.jl [hunk 3]
```diff
@@ -892,9 +927,9 @@ function auto_arima(y::Vector{Fl};
     @assert max_order > 0
     @assert information_criteria in ["aic", "aicc", "bic"]
```
Comment (guilhermebodin) [inline src/models/sarima.jl:928]: Add asserts for tests

**Review by guilhermebodin — CHANGES_REQUESTED** [review]

File: src/models/sarima.jl (hunk 1 above)
Comment (iagochavarry) [reply src/models/sarima.jl:918]: There is!

**Pull request merged**